frontend/apps/reader/readerui.lua

```diff
@@ -465,18 +465,23 @@ end
 function ReaderUI:notifyCloseDocument()
     self:handleEvent(Event:new("CloseDocument"))
     if self.document:isEdited() then
-        UIManager:show(ConfirmBox:new{
-            text = _("Do you want to save this document?"),
-            ok_text = _("Yes"),
-            cancel_text = _("No"),
-            ok_callback = function()
-                self:closeDocument()
-            end,
-            cancel_callback = function()
-                self.document:discardChange()
-                self:closeDocument()
-            end,
-        })
+        if G_reader_settings:readSetting("disable_save_document") then
```

Comment:
From my own opinion, this function is not really useful. Currently only PdfDocument implemented this function, but it writes highlight to the original file. Considering highlight is also stored in metadata file, and as a reader, I believe in most cases, we discard changes.